### ISSUE TITLE
TFLite conversation failed

### ISSUE THREAD

**Issue by smitshilu** [tensorflow/tensorflow#19005]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MAC OSX
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: 1.7
- **Python version**: 3.6
- **Bazel version (if compiling from source)**: 0.11.0
- **CUDA/cuDNN version**: 9/7
- **GPU model and memory**: 1080Ti 12G
- **Exact command to reproduce**: N/A

I wrote a code for mobile net and generated model. Converted model from keras(.h5) to frozen graph(.pb) Summary of graph is 
```
$ ./bazel-bin/tensorflow/tools/graph_transforms/summarize_graph   --in_graph=s.pb
Found 1 possible inputs: (name=input_1, type=float(1), shape=[?,224,224,3]) 
No variables spotted.
Found 1 possible outputs: (name=dense_1/Sigmoid, op=Sigmoid) 
Found 4254887 (4.25M) const parameters, 0 (0) variable parameters, and 4 control_edges
Op types used: 164 Switch, 154 Const, 84 Mul, 56 Add, 28 Sub, 28 Merge, 27 Relu, 27 Minimum, 27 FusedBatchNorm, 27 Rsqrt, 15 Conv2D, 14 Pad, 13 DepthwiseConv2dNative, 3 Shape, 2 BiasAdd, 2 StridedSlice, 2 Pack, 2 Reshape, 2 RealDiv, 1 Max, 1 RandomUniform, 1 MatMul, 1 Floor, 1 Sigmoid, 1 Mean, 1 Exp, 1 Sum, 1 Placeholder
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=s.pb --show_flops --input_layer=input_1 --input_layer_type=float --input_layer_shape=-1,224,224,3 --output_layer=dense_1/Sigmoid
```
When I try to convert it into tflite it gives me following error:
```
$ ./bazel-bin/tensorflow/contrib/lite/toco/toco --input_file=s.pb --input_format=TENSORFLOW_GRAPHDEF  --output_format=TFLITE --output_file=train-1.tflite --inference_type=FLOAT --input_arrays=input_1 --output_arrays=dense_1/Sigmoid --v=2
2018-05-01 14:13:50.005925: I tensorflow/contrib/lite/toco/import_tensorflow.cc:1236] Converting unsupported operation: RandomUniform
2018-05-01 14:13:50.233921: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] Before Removing unused ops: 640 operators, 986 arrays (0 quantized)
2018-05-01 14:13:50.454878: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] After Removing unused ops pass 1: 639 operators, 984 arrays (0 quantized)
2018-05-01 14:13:50.696498: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] Before general graph transformations: 639 operators, 984 arrays (0 quantized)
2018-05-01 14:13:50.861230: F tensorflow/contrib/lite/toco/graph_transformations/resolve_batch_normalization.cc:86] Check failed: mean_shape.dims() == multiplier_shape.dims() 
Aborted
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
